versions.sh

```diff
@@ -12,7 +12,6 @@ supportedAlpineVersions=(
 )
 defaultDebianSuite="${supportedDebianSuites[0]}"
 declare -A debianSuites=(
-	[11]='' # https://github.com/docker-library/postgres/issues/582 😬
```

Comment:
You should grep for more references to 582 that can be adjusted/removed now :eyes:

(I was going to suggest leaving an example here like we do for `alpineVersions` below, but I think ultimately we should remove all this complex logic instead and hope something like https://github.com/docker-library/postgres/issues/582 doesn't happen again :crossed_fingers:)